engine/app/components/citizens_advice_components/radio_group.html.haml

```diff
@@ -0,0 +1,16 @@
+.cads-form-field{ class: ("cads-form-field--has-error" if error?) }
+  .cads-form-field__error-marker
+  %fieldset.cads-form-field__content.cads-radio-group{ fieldset_attributes }
+    %legend.cads-form-field__label
+      = legend
+      - if optional?
+        %span.cads-form-field__optional= t("citizens_advice_components.input.optional")
+    - if hint?
+      %p.cads-form-field__hint= hint
+    - if error?
+      %p.cads-form-field__error-message= error_message
+    - radio_buttons.each do |radio|
+      .cads-radio-group__item
+        %input.cads-radio-group__input{ radio.attributes(name) }
```

Comment:
Wanted a user to only have to pass `name` once per group - wonder if there is a nicer way of sharing the `name` between classes?